app/src/main/java/com/example/onsalestore/objects/CommentItem.java

```diff
@@ -0,0 +1,28 @@
+package com.example.onsalestore.objects;
+
+import com.parse.ParseClassName;
+import com.parse.ParseObject;
+import com.parse.ParseUser;
+
+import org.json.JSONArray;
+
+@ParseClassName("CommentItem")
+public class CommentItem extends ParseObject {
+    public static final String KEY_USER = "user";
+    public static final String KEY_COMMENT = "comment";
+
+    public CommentItem() {
+    }
+
+    public CommentItem(String message) {
+        setComment(message);
+        setUser();
+    }
+
+    public ParseUser getUser() {return getParseUser(KEY_USER);}
+
+    public void setUser() {ParseUser currentUser = ParseUser.getCurrentUser();put(KEY_USER, currentUser);}
```

Comment:
Should be broken down into lines.
```suggestion
    public void setUser() {
        ParseUser currentUser = ParseUser.getCurrentUser();
        put(KEY_USER, currentUser);
    }
```